book/history.md

```diff
@@ -326,10 +326,11 @@ on, the HTML specification ended up moving to a different standards body called
 the [WHATWG](https://whatwg.org/)\index{WHATWG}, but
 [CSS](https://drafts.csswg.org/) and other features are still standardized at
 the W3C. JavaScript\index{JavaScript} is
-standardized at [TC39](https://tc39.es/)\index{TC39}
-(“Technical Committee 39” at
-[ECMA](https://www.ecma-international.org/about-ecma/history/), yet another
-standards body). [HTTP](https://tools.ietf.org/html/rfc2616) is standardized by
+standardized at yet another standards body,
+[TC39](https://tc39.es/)\index{TC39}
+(“Technical Committee 39”) at
+[ECMA](https://www.ecma-international.org/about-ecma/history/).
+[HTTP](https://tools.ietf.org/html/rfc2616) is standardized by
```

Comment:
This changes
> standardized at TC39 [XX] ("Technical Committee 39" at ECMA, yet another standard body.
to
> standardized at yet another standards body, TC39 ("Technical Committee 39" [XX]) at ECMA.
Sounds better to my ear, avoid ugly typography with parentheses / citations.